docs/usage/configuration/config_documentation.md

```diff
@@ -2370,6 +2370,27 @@ Example configuration:
 enable_metrics: true
 ```
 ---
+### `enable_legacy_metrics` (experimental)
```

Comment:
What's the rough plan here? (I'm not super thrilled about maintaining two sets of metrics exporters in perpetuity)

Reply:
I'm not entirely convinced we should even be documenting this. We don't document other `experimental_features` settings, as a rule.

Reply:
I'd like to get this PR in first of all in order to give a migration path.
Straight after this PR, we fix the Grafana dashboard (in `contrib/` so it uses the modern style), use that on matrix.org and make sure it all fits together. (Rich has pointed out that we have *rules* in Prometheus which will need to be updated to use the right names.)
Then we can announce a deprecation notice, perhaps half-way through change the default, then eventually remove it entirely. (Schedule TBD but it'd be good to have one.)

Maybe I should have just linked you to https://github.com/matrix-org/synapse/issues/11106#issuecomment-1216754026 :-)

Reply:
Yeah, maybe this shouldn't be documented. Was on the fence about it, especially because we have people unhappy with config options that aren't documented who will then complain that it's a result of our config manual :p